pkg/failsaferules/failsaferules.go

```diff
@@ -0,0 +1,73 @@
+package failsaferules
+
+import (
+	"fmt"
+)
+
+type TransportProtoFailSafeRule struct {
+	serviceName string
+	port        uint16
+}
+
+var tcp = []TransportProtoFailSafeRule{
+	{
+		"Kubernetes API",
+		6443,
+	},
+	{
+		"ETCD",
+		2380,
+	},
+	{
+		"ETCD",
+		2379,
+	},
+	{
+		"SSH",
+		22,
+	},
+}
+
+var udp = []TransportProtoFailSafeRule{
+	{
+		"DHCP",
+		68,
+	},
+}
+
+func GetTCP() []TransportProtoFailSafeRule {
+	return tcp
+}
+
+func GetUDP() []TransportProtoFailSafeRule {
+	return udp
+}
+
+func (t TransportProtoFailSafeRule) IsConflict(port uint16) (bool, error) {
```

Comment:
didn't you agree on moving conflict functions below back to the webhook ?

Reply:
I thought you meant removing the interface and not all the type methods.

Reply:
this is common pkg for use by webhook and ds the conflict functions are only used by webhook so I think they better stay there , agree ?

Reply:
Ya, makes sense. Moving them.